src/NUnitFramework/framework/Internal/Tests/TestAssembly.cs

```diff
@@ -39,25 +39,25 @@ public class TestAssembly : TestSuite
     {
         /// <summary>
         /// Initializes a new instance of the <see cref="TestAssembly"/> class
-        /// specifying the Assembly and the path from which it was loaded.
+        /// specifying the Assembly and the name of the assembly.
```

Comment:
`and the name for the test suite` perhaps, to match the parameter documentation?